courses/free-courses-ar.md

```diff
@@ -151,6 +153,9 @@
 
 ### HTML / CSS
 
+* [برمجة المواقع | تعلم لغة الhtml من الصفر](https://www.youtube.com/playlist?list=PLYyqC4bNbCIfMY5CoGmiWaPi9l86qaz5B) - أكاديمية ترميز
+* [برمجة المواقع | سلسلة دروس لغة css](https://www.youtube.com/playlist?list=PLYyqC4bNbCIdES52srHE6xTiIgvgMkBWu) - أكاديمية ترميز
+* [CSS Art Tutorials](https://www.youtube.com/playlist?list=PLuXY3ddo_8nzxCiht69IlCe0_VeIuh4ty) - Codezilla
```

Comment:
`CSS Art Tutorials` this like a challenge CSS playlist, maybe it will be endless, so should I add `(🚧 in process)` label too?